src/_data/projects.yml

```diff
@@ -24,7 +24,7 @@
    img_description: Logo for NATO Resolute Support
    img_credit: U.S. Central Command
    project_description: |
-     Over several months, a small DDS team on the ground in Afghanistan replaced a legacy system used to provide NATO advisors mission-critical information. ANET is a system used by NATO advisors deployed to Afghanistan to track engagements with their Afghan Government counterparts. The legacy system had connectivity and usability flaws hindering the mission. Using modern software development practices, DDS built a faster system with better usability that was fully deployed on a classified network to roughly 800 advisors across Afghanistan. ANET has been handed over to NATO for ongoing development and maintenance. Unclassified source code was also released on <a href="https://www.code.mil">Code.mil</a>.
+      ANET is a system used by NATO advisors deployed to Afghanistan to track engagements with their Afghan Government counterparts. The legacy system had connectivity and usability flaws hindering the mission. Using modern software development practices, a small team of DDS engineers built a faster system with better usability that was fully deployed on a classified network to roughly 800 advisors across Afghanistan. ANET has been handed over to NATO for ongoing development and maintenance. Unclassified source code was also released on <a href="https://www.code.gov">Code.mil</a>.
```

Comment:
Check indentation here is consistent with the other items in this file.

Reply:
Link text mentions Code.mil, but anchor links to Code.gov. Which is correct?

Reply:
It looks like you need to rebase, I made these changes that jgarber caught on my PR and they're in master

Reply:
Mo gave me copy edits to the ANET entry, but otherwise my branch says it is up to date with master. Oddly, it didn't pull in the indent and url edits though, so I updated the below manually to match:

- [x] Fixed url to point to `.mil`
- [x] Matched `project_description:` indentation